src/main/java/redux/api/enhancer/Middleware.java

```diff
@@ -10,7 +10,7 @@
  * @param <S> The store type
  * @see <a href="http://redux.js.org/docs/advanced/Middleware.html">http://redux.js.org/docs/advanced/Middleware.html</a>
  */
-interface Middleware<S> {
+public interface Middleware<S> {
```

Comment:
Not exactly. All interface members are public, but the interface itself was package public, so it can only be accessed by classes with a package of `redux.api.enhancer`.